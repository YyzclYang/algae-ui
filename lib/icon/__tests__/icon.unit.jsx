import React from 'react';
import * as renderer from 'react-test-renderer';
import Icon from '../index';

describe('icon', () => {
  it('name 为 wechat 的 Icon', () => {
    const json = renderer.create(<Icon name="wechat" />).toJSON();
    expect(json).toMatchSnapshot();
  });
});
