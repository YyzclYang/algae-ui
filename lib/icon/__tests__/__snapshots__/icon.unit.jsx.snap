// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon name 为 wechat 的 Icon 1`] = `
<svg
  className="yu-ui-icon"
>
  <use
    xlinkHref="#wechat"
  />
</svg>
`;
